coalib/bears/Bear.py

```diff
@@ -319,7 +338,7 @@ def run_bear_from_section(self, args, kwargs):
                 self.name), str(err))
             return
         if self.debugger:
-            return debug_run(self.run, Debugger(), *args, **kwargs)
+            return debug_run(self.run, Debugger(bear=self), *args, **kwargs)
```

Comment:
Can't changed to `debug_run(self.run, *args, Debugger(bear=self), **kwargs)` for python3.4 during running pytest it will throw SyntaxError `SyntaxError: only named arguments may follow *expression`